core/kazoo_modb/priv/couchdb/views/services.json

```diff
@@ -31,23 +31,34 @@
                 "  var unix = new Date(Math.floor((doc.pvt_created - 62167219200) * 1000));",
                 "  var day = unix.getUTCFullYear() + '-' + ('0' + (unix.getUTCMonth() + 1)).slice(-2) + '-' + ('0' + unix.getUTCDate()).slice(-2);",
                 "  if (doc.pvt_type === 'audit_log') {",
-                "    var invoice = doc.invoice || {};",
-                "    var items = invoice.items || [];",
-                "    for (var index = 0; index < items.length; index++) {",
-                "      var item = items[index];",
-                "      if (item.changes && (item.changes.difference.billable || 0 ) !== 0) {",
-                "        var changedBillable = item.changes.difference.billable;",
-                "        emit([day, item.item], {",
-                "          'addition': changedBillable > 0 ? changedBillable : 0,",
-                "          'last_timestamp': doc.pvt_created,",
-                "          'quantity': item.billable || 0,",
-                "          'removal': changedBillable < 0 ? Math.abs(changedBillable) : 0,",
-                "          'sum_quantity': false",
-                "        });",
+                "      if (doc.audit.changes.source === 'accounts') {",
+                "          var qty = doc.audit.changes.quantity;",
```

Comment:
You can keep the same structure like `item.changes.difference.billable` above when creating the account's audit log so you don't need to handle especially this case.